src/chart/Summary.js

```diff
@@ -44,11 +48,15 @@
                         context.render();
                     }
                 }, _);
-            }
+            }       
         }
         return retVal;
     };
 
+    Summary.prototype.summary_format = function() {
```

Comment:
@mounicarachakonda its been a minute since i have worked on the visualization library but it looks like your overriding a publish param with a empty function?